src/main/java/by/onliner/test_dev/transformer/RetryAnalyzer.java

```diff
@@ -1,4 +1,4 @@
-package by.onliner.retry;
+package by.onliner.test_dev.transformer;
```

Comment:
Use java codename convention according to packages names 
test_dev is not the right name

ALso parts which are needed for the framework rather than tests can be stored in a package called "core" as it's core functionality

Reply:
Renamed test_dev to core